### ISSUE TITLE
Mac: Update the whole article

### ISSUE THREAD

**Issue by ThomDietrich** [openhab/openhab-docs#203]
In parallel to the Windows article. https://github.com/openhab/openhab-docs/issues/180

This is mostly a writing task for someone running openHAB on Windows I believe/hope.

Have a look at the linux article to get some inspiration :)

Possible content:
* Meeting the prerequisites
* Installation
* Upgrade
* Backup
* Differences and restrictions compared to the more common Linux setup (?)
* ...

**Comment by martinvw** [comment]
So this is the issue you were actually referring to, can you assign it to me?

**Comment by ThomDietrich** [comment]
@kaikreuzer could you add martin as a contributor so I can select him as the assignee. If possible you could also give the privileges to do so to me...

**Comment by kaikreuzer** [comment]
@ThomDietrich Done and done!

**Comment by ThomDietrich** [comment]
Thanks Kai!

**Comment by Confectrician** [comment]
Hey @martinvw,

You wanted to be assigned here.
Is there stell need for an update and (if yes) would you still ike to take care of it?

#655 could also be interesting here.

**Comment by martinvw** [comment]
No its not likely that I will pick it up soon, to busy with live and reviewing PR's :-)

So please take it over and/or remove my assignment

**Comment by Confectrician** [comment]
We just had some update for openHAB 3.
If there are still things to do we can pick them up in a new issue.
